### DISCUSSION TITLE
Don't use login and password?

### DISCUSSION THREAD

**Discussion by UrbanSide** [SteamRE/DepotDownloader#455] [Q&A] [answered]
Is it possible to somehow use a ready-made manifest and decryption keys for downloading?

**Comment by psychonic** [comment]
What's the use case? It is extremely rare that they remove manifests from the CDN. Outside of that, manifest content and the depot decryption keys should always be available if your account owns the game.

**Reply by UrbanSide** [reply]
The reason, for example, is to create various launchers that can be used by other people, to whom I, for example, would not want to transfer my data...Or is this not supported here?)

**Reply by psychonic** [reply, marked as answer]
No, piracy is not supported here. The code is intentionally open source though. You're welcome to tweak it to your needs, but that's not something anyone here can assist you with.
